tests/app/service/test_rest.py

```diff
@@ -3645,3 +3649,365 @@ def test_get_returned_letter(admin_request, sample_letter_template):
     assert not response[4]['original_file_name']
     assert not response[4]['job_row_number']
     assert response[4]['uploaded_letter_file_name'] == 'filename.pdf'
+
+
+@pytest.mark.parametrize('channel', ["test", "severe"])
+def test_set_as_broadcast_service_sets_broadcast_channel(
+    client, notify_db, sample_service, broadcast_organisation, channel
+):
+    assert sample_service.service_broadcast_settings is None
+    data = {
+        'broadcast_channel': channel,
+        'service_mode': 'live',
+        'provider_restriction': None,
```

Comment:
I haven't got the admin part of this to check, but could the data ever come through from admin as `"None"` (a string)?

Reply:
So I've got it up and running locally with the admin app doing

```
data = {
            "service_mode": service_mode,
            "broadcast_channel": broadcast_channel,
            "provider_restriction": None
        }

        return self.post("/service/{}/set-as-broadcast-service".format(service_id), data)
```

And the API handles it well.

What I will do though is add a test to make sure that the string `"None"` is rejected by the API

Reply:
Have pushed up a rebased commit with the `"None"` test case added